src/treq/response.py

```diff
@@ -91,3 +93,24 @@ def cookies(self):
                 jar.set_cookie(cookie)
 
         return jar
+
+    def check_status(self):
+        """
+        Fail with an error if the response has an error status.
+
+        An error status is a 4xx or 5xx code (RFC 7231).
+
+        :rtype: A `Deferred` that fires with the response itself if
+            the status is a known non-error code, or that calls its
+            errback chain with a :class:`twisted.web.error.Error`
```

Comment:
This is nitpicky, so feel free to ignore it, but the errback will actually be called with a Failure that wraps the exception. Perhaps "``... or that fails it with :exc:`twisted.web.error.Error` ``".

Do fix the Sphinx formatting to use `:exc:` instead of `:class:` though.